client/app/services/chargeback.service.js

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable camelcase */
+
+(function() {
+  'use strict';
+
+  angular.module('app.services')
+    .factory('Chargeback', ChargebackFactory);
+
+  /** @ngInject */
+  function ChargebackFactory(lodash) {
+    var service = {
+      adjustRelativeCost: adjustRelativeCost,
+      currentReport: currentReport,
+      processReports: processReports,
+      reportUsedCost: reportUsedCost,
+    };
+
+    // recomputes items[*].chargeback_relative_cost
+    function adjustRelativeCost(items) {
+      var sums = lodash(items)
+        .pluck(['chargeback', 'used_cost_sum'])
+        .sort()
+        .filter(angular.identity) // nonzero
+        .value();
+
+      var len = sums.length;
+      var bounds = [ len * 0.25, len * 0.5, len * 0.75 ];
+
+      items.forEach(function(item) {
+        if (!item.chargeback.used_cost_sum) {
+          item.chargeback_relative_cost = '';
+
+          return;
+        }
+
+        var idx = sums.findIndex(function(v) {
+          return v === item.chargeback.used_cost_sum;
+        });
+        if (idx < bounds[0]) {
+          item.chargeback_relative_cost = '$';
+        } else if (idx < bounds[1]) {
+          item.chargeback_relative_cost = '$$';
+        } else if (idx < bounds[2]) {
+          item.chargeback_relative_cost = '$$$';
+        } else {
+          item.chargeback_relative_cost = '$$$$';
```

Comment:
Assuming `fa-usd` looks the same, I'd go with the dollars, but mostly because I don't have to write a custom component to show the favicon N times..